c10/xpu/XPUFunctions.cpp

```diff
@@ -52,11 +55,18 @@ inline void initGlobalDevicePoolState() {
     TORCH_WARN("XPU device count is zero!");
     return;
   }
-
   // The default context is utilized for each Intel GPU device, allowing the
   // retrieval of the context from any GPU device.
-  gDevicePool.context = std::make_unique<sycl::context>(
-      gDevicePool.devices[0]->get_platform().ext_oneapi_get_default_context());
+  try {
+    gDevicePool.context =
```

Comment:
FYI: please rebase code if [126752](https://github.com/pytorch/pytorch/pull/126752) is landed.

Reply:
hi @min-jean-cho could you help merge https://github.com/pytorch/pytorch/pull/126752 to this PR?

Reply:
Done, thanks.